### DISCUSSION TITLE
Support for static site export

### DISCUSSION THREAD

**Discussion by legout** [zauberzeug/nicegui#645] [Ideas / Feature Requests]
As already discussed on [reddit](https://www.reddit.com/r/nicegui/comments/11wl7op/comment/jdm6pee/?context=3) exporting nicegui apps to (prerendered) static sites would be a great feature, which might enhance SEO and performance, according to 

https://jamstack.org/ or
https://www.freecodecamp.org/news/how-to-make-static-site-dynamic/

nuxt is a web framework for vue.js apps, which allows for static site export. https://nuxt.com/docs/getting-started/deployment#static-hosting

 Is it possible to make nicegui compatible to nuxt?

**Comment by falkoschindler** [comment]
Thanks for bringing this up, @legout!
We have a very similar discussion that was opened yesterday: https://github.com/zauberzeug/nicegui/discussions/637
Would you mind posting these references over there? Than we can merge all insights about this topic and close this one as duplicate. Thanks!

**Comment by legout** [comment]
Posted my comment here https://github.com/zauberzeug/nicegui/discussions/637#discussioncomment-5446216 and will close this discussion.
